package.json

```diff
@@ -28,21 +28,17 @@
     "!**/*.test.ts"
   ],
   "scripts": {
-    "start": "next dev",
-    "build": "npm run build:ts",
+    "dev": "edgespec dev",
+    "build": "npm run build:ts && edgespec bundle -o dist/bundled-routes.js",
     "prebuild": "npm run build:codegen",
     "postbuild": "node ./dist/index.js",
-    "build:codegen": "npm run build:route-types",
-    "prebuild:codegen": "npm run build:nsm",
-    "postbuild:codegen": "npm run build:openapi",
+    "build:codegen": "npm run build:route-types && npm run build:openapi",
     "build:ts": "tsup",
     "prebuild:ts": "del dist",
     "postbuild:ts": "tsup --format cjs src/index.ts",
-    "build:nsm": "nsm build",
-    "build:route-types": "nextlove generate-route-types --packageDir . --outputFile ./src/route-types.ts --apiPrefix / --pathGlob '/src/pages/api/**/*.ts'",
+    "build:route-types": "edgespec codegen route-types -o src/route-types.ts",
     "postbuild:route-types": "prettier --write src/route-types.ts",
-    "build:openapi": "nextlove generate-openapi --packageDir . --outputFile ./public/openapi.json --apiPrefix / --pathGlob '/src/pages/api/**/*.ts'",
-    "prebuild:openapi": "mkdirp public",
+    "build:openapi": "edgespec codegen openapi -o public/openapi.json",
```

Comment:
I wonder if we should even generate this, vs. can we serve it from the fake server? 

See, e.g., https://github.com/seamapi/javascript-http/blob/main/test/seam/connect/openapi.test.ts#L11

Reply:
interesting... I'd be ok generating it on request in the dev server but including that logic in the bundle may bloat it by quite a bit
not sure how heavy the openapi builder package is